cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -149,7 +151,8 @@ private void ensureTopicsCreated(Map<String, ?> config) {
         throw new IllegalStateException(String.format("Kafka cluster has no alive brokers. (zookeeper.connect = %s",
                                                       config.get(KafkaCruiseControlConfig.ZOOKEEPER_CONNECT_CONFIG)));
       }
-      int replicationFactor = Math.min(2, numberOfBrokersInCluster);
+      int replicationFactor = _metricSampleStoreTopicReplicationFactor == null ? Math.min(2, numberOfBrokersInCluster) :
```

Comment:
Should we avoid hardcoding values in code? -- e.g. `2` --> can we move it to a variable `DEFAULT_...`?